source/documentation/custom_branding.md

```diff
@@ -0,0 +1,26 @@
+GOV.UK Pay supports custom branding on payment pages.
+
+You can [contact the GOV.UK Pay](/support_contact_and_more_information) to
+request customisation for the following features:
+
+* the logo displayed in the left-hand side of the top banner
+* the background and border colour of the top banner
+* your payment confirmation email from GOV.UK Notify
```

Comment:
@katiebates @karlchillmaid @mpwoods is there somewhere we can link to about this guidance?

Reply:
To request custom branding for Notify email templates you need to be signed in to your account. In _Settings_ there's a field called _Email branding_ where you can choose the type of customisation you want from a list of available options. This sends a request to the Notify team, who contact the user directly to request logos etc.
At the moment there isn't any detailed information available when signed out of Notify.
Branding is mentioned on the _Features_ page, but it's probably not informative enough to link to from here.

Reply:
OK, thanks @karlchillmaid. Unless we have some further guidance I can link to I'm a little bit hesitant to include that line in this documentation unless we can be assured that people will already have enough context about the subject by this point.